app/ui/legacy/src/main/java/com/fsck/k9/navigation/ToSetupAccountNavigatorImpl.kt

```diff
@@ -9,4 +10,8 @@ class ToSetupAccountNavigatorImpl : ToSetupAccountNavigator {
     override fun moveToSetupAccount(context: Context, username: String, rootDomain: String?) {
         context.run { AccountSetupBasics.startActivityWithEmailSet(this, username, rootDomain) }
     }
+
+    override fun moveToManualAccountSetup(context: Context) {
+        context.run { startActivity(Intent(this, AccountSetupBasics::class.java)) }
```

Comment:
why not to just pass the context directly instead of making it a receiver?

Reply:
I just did not want to repeat myself
```Kotlin
context.startActivity(Intent(context, .....)
```
But I will change it to 
```Kotlin
    override fun moveToSetupAccount(context: Context, username: String, rootDomain: String?) {
        AccountSetupBasics.startActivityWithEmailSet(context, username, rootDomain)
    }

    override fun moveToManualAccountSetup(context: Context) {
        context.startActivity(Intent(context, AccountSetupBasics::class.java))
    }
```
because in `moveToSetupAccount` I can omit `run` call